Dockerfile

```diff
@@ -5,6 +5,11 @@ ENV BUILD_NUMBER=${BUILD_NUMBER:-1_0_0}
 
 WORKDIR /app
 ADD . .
+
+ADD --chown=appuser:appgroup https://truststore.pki.rds.amazonaws.com/global/global-bundle.pem ./global-bundle.pem
+COPY ./split-pem.bash split-pem.bash
+RUN ./split-pem.bash global-bundle.pem aws-certs.jks
```

Comment:
turns out that sql server can't just use a pem file as is so need to convert into a keystore first